lib/kms.js

```diff
@@ -6,6 +6,31 @@ import * as brHttpsAgent from 'bedrock-https-agent';
 import {getAppIdentity} from 'bedrock-app-identity';
 import {KeystoreAgent, KmsClient} from '@digitalbazaar/webkms-client';
 
+const {util: {BedrockError}} = bedrock;
+
+bedrock.events.on('bedrock.init', async () => {
```

Comment:
Doesn't need to be `async`:

```suggestion
bedrock.events.on('bedrock.init', () => {
```